block-management/block-holder.js

```diff
@@ -34,7 +34,8 @@ export default class BlockHolder extends React.Component<PropsType, StateType> {
 
 	getBlockForType() {
 		if ( this.props.blockType === 'code' ) {
-			const Code = codeBlock.edit;
+			const blockType = getBlockType( 'core/code' );
```

Comment:
Maybe we should just directly define 'code/code' in the sample in `block-manager.js`. I haven't checked but what kind of blockType strings will we get after the parsing step?

Reply:
Yes, I think we should use the `core/code` blocktype name directly in the block list array. I can make the change in this PR (let me know!) unless we wait for the rest of the (hardcoded) blocks in the list to be loaded this way first.

> ... what kind of blockType strings will we get after the parsing step

Not sure about the parsing step but from what I see from the block definitions in the GB library, all current blocks are in the form `code/<name>`.

Reply:
Ok, let's do the change in this repo and drop the `if ( this.props.blockType === 'code' ) {` here

Reply:
Wait, we still need some form of the conditional since only the Code block is currently loaded that way... the rest of the blocks in the hardcoded list still need to fall back to `View`s.

Reply:
Not sure to understand, can't we do this?

```diff
diff --git a/block-management/block-holder.js b/block-management/block-holder.js
index b7c11f9..1b91c49 100644
--- a/block-management/block-holder.js
+++ b/block-management/block-holder.js
@@ -33,12 +33,11 @@ export default class BlockHolder extends React.Component<PropsType, StateType> {
        }

        getBlockForType() {
-               if ( this.props.blockType === 'code' ) {
-                       const blockType = getBlockType( 'core/code' );
-                       const Code = blockType.edit;
-                       // TODO: input text needs to be kept by updating the attributes
+               const blockType = getBlockType( this.props.blockType );
+               if ( blockType ) {
+                       const Block = blockType.edit;
                        return (
-                               <Code
+                               <Block
                                        attributes={ { content: this.props.content } }
                                        setAttributes={ attrs => console.log( { attrs } ) }
                                />
```

Reply:
We can, the `if ( blockType ) {` is what I was referring to as "some form of the conditional` :). That's what I'm about to do so, 👍.

Reply:
Done with d55d521.